diff --git a/build/server/redirect-back.d.ts b/build/server/redirect-back.d.ts
index b96ca64aaf202b4103099dc4299e09310566f341..9b8b85276fe07283f8e2947ca752b67128a1e827 100644
--- a/build/server/redirect-back.d.ts
+++ b/build/server/redirect-back.d.ts
@@ -1,3 +1,4 @@
+import { TypedResponse } from "@remix-run/node";
 /**
  * Create a new Response with a redirect set to the URL the user was before.
  * It uses the Referer header to detect the previous URL. It asks for a fallback
@@ -14,4 +15,4 @@
  */
 export declare function redirectBack(request: Request, { fallback, ...init }: ResponseInit & {
     fallback: string;
-}): Response;
+}): TypedResponse<never>;
